src/main/service/GeocodingService.ts

```diff
@@ -0,0 +1,40 @@
+import nodeGeocoder, { Entry, Geocoder, Options } from 'node-geocoder';
+import { GeocodingObject } from '../data/GeocodingObject';
+import * as dotenv from 'dotenv';
+
+/**
+ * A Node library for geocoding our restaurant addresses
+ */
+export class GeocodingService {
+  private geocoder: Geocoder;
+
+  constructor() {
+    dotenv.config();
+    const options: Options = {
+      provider: 'locationiq',
+      apiKey: process.env.GEOCODING_KEY,
+    };
+    this.geocoder = nodeGeocoder(options);
+  }
+
+  public getGeoCoordinatesFromAddress(
```

Comment:
Some documentation for this function if you don't mind :)

Also could we use await and async, no need for catch since errors will be caught at controller level